src/main/java/com/softserve/demo/model/Provider.java

```diff
@@ -64,4 +64,8 @@
     @JoinColumn(name = "location_id")
     private Location location;
 
+    @ManyToOne
```

Comment:
What's the intent of this mapping? Logically provider can have many customers, so as customer can have many providers, no?